### PR TITLE
start docs page with example use cases (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@ehiggs: please review before I flesh out the other use cases?

**Comment by boegel** [comment]
preview at http://boegel-hod.readthedocs.org/en/example_use_cases/Example_use_cases.html

**Comment by ehiggs** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
@ehiggs: I think this is ready, see http://boegel-hod.readthedocs.org/en/example_use_cases/Example_use_cases.html

File: docs/Example_use_cases.rst [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+.. _example_use_cases_common_configuring_hod:
+
+Configuring HOD
+~~~~~~~~~~~~~~~
+
+
+You can/should configure HOD by defining the HOD work directory and specifying which module should be loaded
+in the HOD job being submitted (see also :ref:`cmdline_create_options`)::
+
+    # for 'hod batch'
```
Comment (ehiggs) [inline docs/Example_use_cases.rst]: Remove comment and make this a heading for the code segment.

Mention that these could automatically be set if they define it in an easybuild file.

File: docs/Example_use_cases.rst [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+
+See the example below for more details; basic usage information for ``hod create`` is available at :ref:`cmdline_create`.
+
+.. _example_use_cases_interactive_hadoop_screen:
+
+Using ``screen``
+~~~~~~~~~~~~~~~~
+
+To interactively start commands that may require some time to finish, we strongly recommended starting a
+so-called *screen* session after connecting to the HOD cluster.
```
Comment (ehiggs) [inline docs/Example_use_cases.rst]: Link screen to [here](https://www.gnu.org/software/screen/).

Comment (boegel) [inline docs/Example_use_cases.rst]: There's a link below to the screen user manual

**Pull request merged**